docs/NewAppointment.puml

```diff
@@ -0,0 +1,29 @@
+@startuml
+'https://plantuml.com/sequence-diagram
+
```

Comment:
Neat and tidy, very detailed.

You may consider adding colours and hide the footbox 👍